diff --git a/tests/decode.sh b/tests/decode.sh
index 59ad7cc..d1df62b 100755
--- a/tests/decode.sh
+++ b/tests/decode.sh
@@ -1,8 +1,8 @@
-#!/bin/bash -eu
+#! @bash@/bin/bash -eu

 hsTmpDir=$1

-ghc                                         \
+@ghc@/bin/ghc                               \
     --make                                  \
     -odir $hsTmpDir                         \
     -hidir $hsTmpDir                        \
diff --git a/tests/encode.sh b/tests/encode.sh
index 8ebea7e..bc79de2 100755
--- a/tests/encode.sh
+++ b/tests/encode.sh
@@ -1,8 +1,8 @@
-#!/bin/bash -eu
+#! @bash@/bin/bash -eu

 hsTmpDir=$1

-ghc                                         \
+@ghc@/bin/ghc                               \
     --make                                  \
     -odir $hsTmpDir                         \
     -hidir $hsTmpDir                        \
